src/Symfony/Bundle/FrameworkBundle/Resources/config/mailer_debug.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\DependencyInjection\Loader\Configurator;
+
+use Symfony\Component\Mailer\DataCollector\MessageDataCollector;
+
+return static function (ContainerConfigurator $container) {
+    $container->services()
+        ->set('mailer.data_collector', MessageDataCollector::class)
+            ->args([
+                service('mailer.logger_message_listener'),
+            ])
+        ->tag('data_collector', [
+            'id' => 'mailer',
+            'template' => '@WebProfiler/Collector/mailer.html.twig',
```

Comment:
please change the order of template and id to be consistent with the other data_collector tags

Reply:
✅